en/docs/integrations/build-your-first-integration.md

```diff
@@ -1,59 +1,133 @@
+
+        | **Field**      | **Value**                     |
+        |----------------|-------------------------------|
+        | **Type**       | `int`                         |
+        | **Name**       | `population`                  |
+        | **Expression** | `(populationByCountry is worldbank:CountryPopulationArr ? populationByCountry[0]?.value ?: 0 : 0) / 1000000`       |
+
+    3. Click **Save**.
+
+## Step 4: Calculate the total COVID-19 case count by population 
+
+Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
+
+1. Click the last **+** icon in the low-code diagram.
+2. Under **Statements**, click **Variable** and enter details as follows:
+
+    | **Field**      | **Value**                     |
+    |----------------|-------------------------------|
+    | **Type**       | `var`                         |
+    | **Name**       | `totalCasesPerMillion`        |
+    | **Expression** | `totalCases / population`     |
+
+3. Click **Save**.
+         
+## Step 5: Generate and send an email with summary statistics
+Follow this procedure to send an email with the summary of statistics to a specified email address:
+
+1. Click the last **+** icon in the low-code diagram.
+2. Under **Statements**, click **Variable** and enter details as follows:
+
+    | **Field**      | **Value**                     |
+    |----------------|-------------------------------|
+    | **Type**       | `string`                      |
+    | **Name**       | `mailBody`                    |
+    | **Expression** | `"Per Million Data: Total Cases" + totalCasesPerMillion.toString()`|
+
+3. Click **Save**.
+4. Click the last **+** icon in the low-code diagram.
+5. Click **API Calls** and then click **Mail by Choreo**.
+6. In the **Mail by Choreo Connection** window, enter details as follows: 
+
+    1. In the **Operation** drop-down list, select `Send Mail`.
+    2. In the **Recipient** field, enter the email address to which you want to send the email. For example, `"test@gmail.com"`
+    3. In the **Subject** field, enter `"Total COVID -19 Cases in the USA"`
+    4. In the **Message Body** drop-down list, select `mailBody`.
+
+7. Click **Save**.
+
+Now you have successfully created and configured the integration. It looks as follows:
+
+- In the low-code view
+
+    ![Low-code view](../assets/img/integrations/low-code-view.png){.cInlineImage-full}
+
+- In the code view
+  
+    ![Code view](../assets/img/integrations/code-view.png){.cInlineImage-full}
+
+## Step 7: Try out the integration
+
+1. Click **Run & Test**. This starts the integration and sends an email to the recipient you specified when configuring the integration.
+2. Go to the inbox of the email recipient and take a look at the COVID-19 statistics summary mail. 
+
+     ![Summary email](../assets/img/integrations/summary-email.png){.cInlineImage-full}
```

Comment:
Can we crop the image as highlighted below. To avoid `External` and `Send to me` labels.

<img width="829" alt="image" src="https://user-images.githubusercontent.com/16300038/122114160-9d774400-ce40-11eb-8c61-3e0c2950f49f.png">

Reply:
Updated.